dali/driver/uart.py

```diff
@@ -0,0 +1,109 @@
+
+"""
+UART communication driver for DALI
+"""
+
+import logging
+import serial
+import struct
+from dali.command import Command
+from dali.interface import DriverInterface, ParameterError
+
+
+class DaliUART(DriverInterface):
```

Comment:
This is a good idea.  For example, the serial driver I committed a few days ago (completely untested!) implements the Tridonic SCI2 protocol.  What's the name of the protocol yours implements?